pygpu/tests/test_elemwise.py

```diff
@@ -101,87 +100,6 @@ def ielemwise2_ops_array(op, dtype1, dtype2, shape):
     assert numpy.allclose(out_c, numpy.asarray(out_g), atol=1e-6)
 
 
-def test_elemwise_layouts():
```

Comment:
Now that I look at it in detail, it's almost all tested, but with smaller arrays.  I'll add a test for an offset, but it's all that's missing.  It might look like more things are tested but they are irrelevant and where only there because I figure I might as well use all the options of gen_gpuarray().

Now that I've seen that it makes for extra-long tests, I tend to be more minimalist.